docs/siem/detections/alerts-ui-manage.asciidoc

```diff
@@ -0,0 +1,106 @@
+
+TIP: Use the icons in the upper left corner of the Alerts table to customize
+displayed columns and row renderers, and view the table in full screen mode.
+
+[float]
+[[detection-alert-status]]
+=== Change alert statuses
+
+You can set an alert's status to indicate whether it needs to be investigated
+(`Open`), is under active investigation (`In progress`), or resolved
+(`Closed`). By default, the Alerts table displays open alerts. To view alerts
+with other statuses, click *In progress* or *Closed*.
+
+To change alert statuses, either:
+
+* In the alert's row, click the *more options* icon, and then select the
+required status (*Mark in progress*, *Close alert*, or *Open alert*).
+* In the Alerts table, select all the alerts you want to change, and then select
+*Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
+
+[float]
+[[signals-to-timelines]]
+=== Send alerts to Timeline
+
+To view an alert in Timeline, click the *Investigate in timeline* icon.
+
+TIP: When you send an alert generated by a
+<<rules-ui-create, threshold rule>> to Timeline, all matching events are
+listed in the Timeline, even ones that did not reach the threshold value. For
+example, if you have an alert generated by a threshold rule that detects 10
+failed login attempts, when you send that alert to Timeline all failed login
+attempts detected by the rule are listed.
+
+If the rule that generated the alert uses a Timeline template, when you
+investigate the alert in Timeline, the dropzone query values defined in the
+template are replaced with their corresponding alert values.
+
+// * `host.name`
+// * `host.hostname`
+// * `host.domain`
+// * `host.id`
+// * `host.ip`
+// * `client.ip`
+// * `destination.ip`
+// * `server.ip`
+// * `source.ip`
+// * `network.community_id`
+// * `user.name`
+// * `process.name`
+
+*Example*
+
+The Timeline template used in the rule has this dropzone query:
+`host.name: "Linux-LiverpoolFC"`. When alerts generated by the rule are
+investigated in Timeline, the `host.name` value is replaced with the alert's
+`host.name` value. If the alerts's `host.name` value is `Windows-ArsenalFC`,
+the Timeline dropzone query is `host.name: "Windows-ArsenalFC"`.
+
+NOTE: See <<timelines-ui>> for information on creating Timelines and Timeline
+templates. For information on how to add Timeline templates to rules, see
+<<rules-ui-create>>.
+
+[float]
+[[add-exception-from-alerts]]
+=== Add rule exceptions
+
+You can add exceptions to the rule that generated the alert directly from the
+Alerts table. Exceptions prevent a rule from generating alerts even when its
+criteria are met.
+
+To add an exception, click the actions icon (three dots) and then select
+_Add exception_. 
+ 
+For information about exceptions and how to use them, see
+<<detections-ui-exceptions>>.
+
+[float]
+[[alerts-to-resolver]]
+=== Visually analyze process relationships.
+
+For process events received from the Elastic Endpoint agent, you can open a
```

Comment:
We might have to loop around and rename this the Elastic Security Endpoint integration. Technically speaking, the Elastic Endpoint Security agent in the legacy Endgame agent, whereas the Elastic agent ingested onto a device is configured with the Elastic Security Endpoint integration.

Reply:
Yep - and we'll need to add the link as well.